multi-history-replay/replayer/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/temporalio/samples-go/helloworld"
+	"go.temporal.io/api/enums/v1"
+	"go.temporal.io/api/history/v1"
+	"go.temporal.io/api/workflow/v1"
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/worker"
+)
+
+func GetWorkflowHistory(ctx context.Context, client client.Client, id, runID string) (*history.History, error) {
+	var hist history.History
+	iter := client.GetWorkflowHistory(ctx, id, runID, false, enums.HISTORY_EVENT_FILTER_TYPE_ALL_EVENT)
+	for iter.HasNext() {
+		event, err := iter.Next()
+		if err != nil {
+			return nil, err
+		}
+		hist.Events = append(hist.Events, event)
+	}
+	return &hist, nil
+}
+
+// GetWorkflows calls ListWorkflow with query and gets as many workflow executions until there are no more or we the number exceeds maxWorkflows
+func GetWorkflows(ctx context.Context, c client.Client, query string, maxWorkflows int) ([]*workflow.WorkflowExecutionInfo, error) {
+	var nextPageToken []byte
+	workflowExecutions := make([]*workflow.WorkflowExecutionInfo, 0)
```

Comment:
```suggestion
	var workflowExecutions []*workflow.WorkflowExecutionInfo
```

Doesn't really matter I just happen to like this one better. But it is subjective (perf diff negligible), so your choice.